frontend/drivers/platform_unix.c

```diff
@@ -1222,22 +1222,8 @@ static enum frontend_powerstate frontend_unix_get_powerstate(
    *percent = battery_level;
 
    ret = (enum frontend_powerstate)powerstate;
-#elif defined(DINGUX)
-   /* Dingux seems to have limited battery
```

Comment:
What about the DINGUX codepath herre? Why is it suddenly gone and only RETROFW remains?

Reply:
Good catch. Squashed with an unrelated commit. Fixed now